### ISSUE TITLE
Gtk-Message: 12:47:16.019: GtkDialog mapped without a transient parent. This is discouraged

### ISSUE THREAD

**Issue by sebastinas** [pwmt/zathura#57]
*On GitLab by @sanjay_ankur on Apr 26, 2018, 14:53*

---

I get this warning on the terminal each time I open the print dialog. Not sure what it's about, but thought it may be worth mentioning here.

```
rpm -qa \*zathura\*
zathura-0.3.9-2.fc28.x86_64
zathura-djvu-0.2.8-1.fc28.x86_64
zathura-pdf-poppler-0.2.9-1.fc28.x86_64
zathura-ps-0.2.6-1.fc28.x86_64
zathura-cb-0.1.8-1.fc28.x86_64
zathura-plugins-all-0.3.9-2.fc28.x86_64
```

**Comment by sebastinas** [comment]
Fixed in 3a68be9a9b3d0d2094b8d3537ce9bdacb1ae5624.

**Comment by sebastinas** [comment]
closed
